.osdk-scorecard.yaml

```diff
@@ -9,11 +9,13 @@ scorecard:
           - "deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml"
           - "deploy/crds/app.kiegroup.org_v1alpha1_kogitoinfra_cr.yaml"
           - "deploy/crds/app.kiegroup.org_v1alpha1_kogitojobsservice_cr.yaml"
+          - "deploy/crds/app.kiegroup.org_v1alpha1_kogitomgmtconsole_cr.yaml"
     # `olm` tests configured to test 2 CRs
```

Comment:
Once we have the 0.9.0 release PR, this should be gone with a rebase. :)

Reply:
What do you mean? Do you want me to undo this change?

Reply:
yes please since it's not related to your change.

Reply:
Okis. Reverted now.